jvb/src/main/kotlin/org/jitsi/videobridge/message/BridgeChannelMessage.kt

```diff
@@ -338,6 +379,12 @@ class ForwardedEndpointsMessage(
  * TODO: consider and adjust the format of videoConstraints. Do we need all of the VideoConstraints fields? Document.
  */
 class SenderVideoConstraintsMessage(val videoConstraints: VideoConstraints) : BridgeChannelMessage(TYPE) {
+    /**
+     * Serialize manually because it's faster than either Jackson or json-simple. Note that kotlin raw strings (triple
+     * quotes) are also significantly slower.
+     */
+    override fun toJson(): String = "{\"colibriClass\":\"$TYPE\", \"videoConstraints\":$videoConstraints}"
```

Comment:
Add a comment to `VideoConstraints.toString()` that we depend on the fact that it returns valid JSON?